guide/interactions/registering-slash-commands.md

```diff
@@ -0,0 +1,192 @@
+# Registering slash commands
+
+Discord provides users the option to create client-integrated slash commands. In this section you'll be learning how to register these commands using discord.js!
```

Comment:
```suggestion
Discord provides users with the option to create client-integrated slash commands. In this section, you'll be learning how to register these commands using discord.js!
```

Reply:
feels off with a comma there, there's no need for a pause bewteen `section` and `you'll` imo

Reply:
Might be worth rephrasing that part entirely tho, not alto happy with it